libs/ui/menu/src/lib/menu/menu.component.scss

```diff
@@ -1,4 +1,5 @@
 @import '~@terminus/ui-styles/helpers';
+@import '../../../../../ui/styles/src/lib/style-helpers/controls/menu';
```

Comment:
Still need to figure out:

- If this is necessary or if it is an issue with storybook
- If we can clean this up with an alias